index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
```

Comment:
Please include the Linters Validation and the required structure:
-include the language in the HTML tag.
-include the Meta tags inside the Head Tag.
-include The Title tag inside the Head Tag.

Reply:
When we are reviewing the code written by students, we want to deliver a clear and specific message. In this comment, you suggested to the student multiple requirements which are not related. As linter validation is not something that relates to a specific line of code, we should address it in the box where we tell to the student if the project is approved or not. :sun_behind_small_cloud: 

Related to the required structure of `html` tag and of `head` tag, we can include examples to be more specific, we can provide links. And always we should explain ourselves. Why do we suggest a particular thing to the student? :sun_with_face: 

[Here is a link about `head` tag](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML) that we can provide to the student for more clarity. 

- [ ] Please update the comments to be more specific regarding the `html` and `head` tags and please provide useful links. :+1: 

- [ ] Kindly, remove the remark about linters from this comment but don't forget to include it in the general box, where we tell to the student if we approved his project or not. :sunglasses: